### PR TITLE
Neo4J database with user creation, project creation, and project ownership

### PR DESCRIPTION
**Pull request by tch1001**
Added connections with neo4j database. 
One can create users and projects with GET requests (will be changed to POST eventually, now it remains open to GET for easier debugging)
Also wrote tests for database objects

### CODE REVIEW

**Comment by bushshrub** [comment]
Did tests pass  yet @tch1001

**Comment by tch1001** [comment]
yes tests pass @xiurobert

File: twig_server/database/Project.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+from twig_server.database.connection import Neo4jConnection, Neo4jDriver
+from twig_server.database.native import Node, Relationship
+from twig_server.database.User import User
+from typing import Any, List, Optional
+from neo4j import Record
+
+
+class Project(Node):
+    _label_name = "Project"
+    _label_owner_relationship = "Project_Owner"
```
Comment (bushshrub) [inline twig_server/database/Project.py:10]: Probably should be a constructor option in the superclass Node

File: twig_server/database/Project.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+    def __init__(
+        self,
+        conn: Neo4jConnection,
+        **kwargs: Any,
+    ) -> None:
+        """Creates/Queries a Project Node
+        :param conn:
+            Neo4J connection
+        :Keyword Arguments:
```
Comment (bushshrub) [inline twig_server/database/Project.py:20]: That's not how RST comments work but ok

File: twig_server/database/Resource.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+from twig_server.database.connection import Neo4jConnection
+from twig_server.database.native import Node, Relationship
+
+
+class Resource(Node):
+    _label_name = "Resource"
+
+    def __init__(
+        self,
+        conn,
```
Comment (bushshrub) [inline twig_server/database/Resource.py]: annotations

**Review by bushshrub — COMMENTED** [review]
Also you need to delete ds store

File: twig_server/database/Project.py (hunk 1 above)
Comment (tch1001) [reply twig_server/database/Project.py:10]: let's just leave it like that for now, i plan to refactor the code when i have a clearer idea of what needs to be done on the server side anyway

**Pull request merged**